src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -534,12 +548,11 @@ export default {
 			if (this.showArchived) {
 				return t('spreed', 'You have no archived conversations.')
 			}
-			switch (this.isFiltered) {
-			case 'mentions':
+			if (this.filters.includes('mentions')) {
```

Comment:
```suggestion
			if (this.filters.length === 1 && this.filters.includes('mentions')) {
```
Otherwise several filters are applied, and you can not say, what user does not have only "...mentions", but not "...messages"